src/opentype/tables/simple/other/vmtx.js

```diff
@@ -38,7 +39,7 @@ class vmtx extends SimpleTable {
 class LongVertMetric {
   constructor(h, b) {
     this.advanceHeight = h;
-    this.topSideBearing = b;
+    this.tsb = b;
```

Comment:
Note that this needs to stay `topSideBearing`, as that's what it's called [in the spec](https://learn.microsoft.com/en-us/typography/opentype/spec/vmtx). However, the class does need to be renamed to `LongVerMetric` because that "t" should not be there.

Reply:
It's so strange that the abbreviated `lsb` is used in `hmtx`.

Reply:
OpenType is more of a journey than a single spec, new tables got added over the years, and not all of them used what previous tables already used (or, intentionally went "that was a bad convention, we're now doing it this way").